docs/src/main/asciidoc/sending-emails.adoc

```diff
@@ -276,11 +276,6 @@ quarkus.mailer.username=YOUREMAIL@gmail.com
 quarkus.mailer.password=YOURGENERATEDAPPLICATIONPASSWORD
 ----
 
-== Using SSL with native executables
-
-Note that if you enable SSL for the mailer and you want to build a native executable, you will need to enable the SSL support.
-Please refer to the native-and-ssl-guide.html[Using SSL With Native Executables] guide for more information.
```

Comment:
I think this is still interesting as you might have to bundle the SSL libraries in your Docker image for instance.

Reply:
So we keep this part?

Reply:
Yes, I would keep it.